implementations/micrometer-registry-humio/src/main/java/io/micrometer/humio/HumioConfig.java

```diff
@@ -56,13 +56,11 @@ default String uri() {
      * datasource defined by these tags.
      * @return Tags which uniquely determine the datasource to store metrics in.
      */
-    @Nullable
-    default Map<String, String> tags() {
+    @Nullable default Map<String, String> tags() {
```

Comment:
I wonder why this didn't end up after `default` but it did other places.

Reply:
I think the formatter does this and the recipe you used uses the other order?

Reply:
But in this same file, the other change place (`apiToken`) ended up the other way